src/pagination/pagination.config.ts

```diff
@@ -8,18 +8,37 @@ export class PaginationConfig {
     maxSize: void 0,
     itemsPerPage: 10,
     boundaryLinks: false,
+    accessibleLinks: false,
+    boundaryAccessibleLinks: false,
     directionLinks: true,
     firstText: 'First',
     previousText: 'Previous',
     nextText: 'Next',
     lastText: 'Last',
+    previousAccessibleText: 'Previous',
```

Comment:
10+ new inputs for accessibility? I think that text for labels should be constructed dynamically from what we have